applications/test/package.scala

```diff
@@ -45,5 +45,7 @@ class ApplicationsTestSuite
       new InteractivePickerTest,
     )
     with SingleServerSuite {
-  override lazy val port: Int = 19003
+
+//  override implicit lazy val portNumber: PortNumber = PortNumber(19003)
```

Comment:
I was expecting this change to be working like it does for other test suites, e.g. `admin/test/package.scala` but all tests are passing only if it's completely removed.

Reply:
Example of tests failing: https://github.com/guardian/frontend/blob/main/common/test/model/ContentTest.scala. They seem to be Selenium tests